src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -758,4 +809,6 @@ export class CourseOverviewComponent implements OnInit, OnDestroy, AfterViewInit
         this.isNavbarCollapsed = !this.isNavbarCollapsed;
         localStorage.setItem('navbar.collapseState', JSON.stringify(this.isNavbarCollapsed));
     }
+
+    protected readonly String = String;
```

Comment:
I do not understand what is happening here, can you explain it to me?

Reply:
something which got in while resolving comments from before